swiperefresh/src/main/java/com/google/accompanist/swiperefresh/SwipeRefresh.kt

```diff
@@ -46,6 +46,8 @@ import kotlinx.coroutines.CoroutineScope
 import kotlinx.coroutines.launch
 import kotlin.math.absoluteValue
 
+typealias SwipeRefreshIndicator = @Composable (state: SwipeRefreshState, refreshTrigger: Dp) -> Unit
```

Comment:
Any reason for this? This will become public API which we should minimize. If you feel strongly about having this as public API, it needs kdoc.

Reply:
No, I just forgot removing it, it was there for the previous implementation where we had 2 Indicator params.